compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangInvokableNode.java

```diff
@@ -42,7 +42,7 @@
  */
 public abstract class BLangInvokableNode extends BLangNode implements InvokableNode {
 
-    public static final String DEFAULT_WORKER_NAME = "default";
+    public static final String DEFAULT_WORKER_NAME = "function";
```

Comment:
Unfortunately, there are some more `DEFAULT_WORKER_NAME` constants with "default". Shall we remove them if they are not being used? 
`BLangConstants.java` is one example.

Reply:
Got merged while I was reviewing :). Shall we create an issue for this?

Reply:
@KRVPerera will address this when addressing https://github.com/ballerina-platform/ballerina-lang/issues/23476#issuecomment-783488634. 
Added comment there. https://github.com/ballerina-platform/ballerina-lang/issues/23476#issuecomment-783856672